lib/openam.js

```diff
@@ -214,14 +224,11 @@ OpenAMClient.prototype.validateAccessToken = function (accessToken, realm) {
  */
 OpenAMClient.prototype.getProfile = function (userId, realm, sessionId, cookieName) {
     //noinspection JSValidateTypes
-    return request.get(this.serverAddress + '/json/users/' + userId, {
+    return request.get(this.serverAddress + '/json' + realm + '/users/' + userId, {
```

Comment:
realm should have a default value of `/`, otherwise this will become `/jsonundefined/users`

Reply:
(actually it's probably always called with a value...)

Reply:
intention was it is always called with value, see agent.js